omnibus/config/projects/agent.rb

```diff
@@ -118,16 +118,26 @@
 # Datadog agent
 dependency 'datadog-agent'
 
+python_runtimes = ENV['PYTHON_RUNTIMES'].split(',')
```

Comment:
let's make this a project-level variable with a default value, something like
```
python_runtimes ENV['PYTHON_RUNTIMES'].nil? ? ['2'] : ENV['PYTHON_RUNTIMES'].split(',')
```
having a default value here is more robust and makes possible bootstrapping the build by just invoking `omnibus`

Reply:
To add project-level variables we need to change https://github.com/DataDog/omnibus-software/blob/master/lib/omnibus-software.rb

I will add the default to the current code.